src/api-doc/Entity Management API's.postman_collection.json

```diff
@@ -0,0 +1,977 @@
+{
+	"info": {
+		"_postman_id": "fa7d1558-cdcb-493a-8767-55758ac735da",
+		"name": "Entity Management API's",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "33513401"
+	},
+	"item": [
+		{
+			"name": "entity",
+			"item": [
+				{
+					"name": "Find",
```

Comment:
@MallanagoudaB use same convention of endpoint , means make it "find". all lower case letters